packages/neuron-wallet/src/services/sync/utils.ts

```diff
@@ -48,4 +48,20 @@ export default class Utils {
     }
     return result
   }
+
+  public static min = (array: bigint[]): bigint | undefined => {
```

Comment:
Why would a numeric operation belong to sync/utils?

Reply:
Utility methods of sync module.